pyproject.toml

```diff
@@ -43,6 +43,7 @@ pytest-rerunfailures = "^13.0"
 mypy = "^1.8.0"
 isort = "^5.13.2"
 flake8 = "^7.0.0"
```

Comment:
Not really relevant to anything (but I do enjoy me some linting topics; and apparently commenting on merged PRs)&hellip;
Reading through the deps reminded me of a recent Locust commit: [locustio/locust#2549](https://github.com/locustio/locust/pull/2549) "Use [ruff](https://docs.astral.sh/ruff/faq/) instead of flake8 and black for linting and formatting".

Anyways... in case you're bored one day and wanted to see how ruff compares to other toolings. 🤷